tools/onnx2tensorrt.py

```diff
@@ -26,11 +26,15 @@ def get_GiB(x: int):
 def _prepare_input_img(img_path: str,
                        test_pipeline: Iterable[dict],
                        shape: Optional[Iterable] = None,
-                       rescale_shape: Optional[Iterable] = None) -> dict:
+                       rescale_shape: Optional[Iterable] = None,
+                       normalize_in_graph: Optional[bool] = False) -> dict:
     # build the data pipeline
     if shape is not None:
         test_pipeline[1]['img_scale'] = (shape[1], shape[0])
     test_pipeline[1]['transforms'][0]['keep_ratio'] = False
+    for transform in test_pipeline[1]['transforms']:
+        if transform['type'] == 'Normalize':
+            transform.normalize_in_graph = normalize_in_graph
```

Comment:
Should explicitly convert input tensor from `uint8` to `float32`
by update line 43:
```
imgs = [img.float() for img in data['img']]